samples/boards/nrf/dynamic_pinctrl/sample.yaml

```diff
@@ -0,0 +1,3 @@
+sample:
+  description: A dynamic pin control sample for nRF board/s.
+  name: dynamic_pinctrl
```

Comment:
Shouldn't `nrf52840dk_nrf52840` be indicated as an integration platform?

Reply:
added